### PR TITLE
improvement for better readability

### PR DESCRIPTION
**Pull request by ghost**
i sorted the commands in the order how they would be used in a used case, i hope this makes sense for others too, please check

### CODE REVIEW

File: docs/installation-and-operations/operation/restoring/README.md [hunk 1]
```diff
@@ -121,21 +121,18 @@ For Docker-based installations, assuming you have a backup as per the procedure
 
 Let's assume you want to restore a database dump given in a file, say `openproject.sql`.
 
+This assumes that the database container is called `db_1`. Find out the actual name on your host using `docker ps | postgres`.
```
Comment (oliverguenther) [inline docs/installation-and-operations/operation/restoring/README.md]:
```suggestion
This assumes that the database container is called `db_1`. Find out the actual name on your host using `docker ps | grep postgres`.
```

File: docs/installation-and-operations/operation/restoring/README.md [hunk 2]
```diff
@@ -121,21 +121,18 @@ For Docker-based installations, assuming you have a backup as per the procedure
...
-5. Source the dump with psql on the container: `docker exec -it db_1 psql -U postgres` followed first by `\c openproject` and then by `\i openproject.sql`. You can leave this console by entering `\q` once it's done.
-6. Delete the dump on the container: `docker exec -it db_1 rm openproject.sql`
-7. Restart the web and worker processes: `docker-compose start web worker`
-
-_\* If your database doesn't have an OpenProject user you either have to create one or use which ever user you used before._
-
-* * *
-
-This assumes that the database container is called `db_1`. Find out the actual name on your host using `docker ps | postgres`.
+3. Create a user/role for the OpenProject database: `docker exec -it db_1 psql -U postgres -c 'create user openproject;'`
```
Comment (oliverguenther) [inline docs/installation-and-operations/operation/restoring/README.md]:
```suggestion
3. If your database doesn't have an openproject user yet, create it with this command: `docker exec -it db_1 psql -U postgres -c 'create user openproject;'`
```

As the user might already be present, we should keep the comment from earlier

Comment (ghost) [reply docs/installation-and-operations/operation/restoring/README.md]: Just an off topic question, I followed this docs that I wanted to edit in this PR: I installed OP via docker-compose and after up -d and down web worker, I dropped the database OpenProject, the user OpenProject was not there afterwards, before I don't know.

**Review by machisuji — APPROVED** [review]
👍

**Pull request merged**